contracts/utils/TokenUtils.sol

```diff
@@ -8,7 +8,7 @@ import "./SafeERC20.sol";
 library TokenUtils {
     using SafeERC20 for IERC20;
 
-    address public constant WETH_ADDR = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public constant WETH_ADDR = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2; //TODO check how to change depending on chain?
```

Comment:
@yippee-ki-yay this is currently a problem, any idea how to make this easier to change?

Reply:
Maybe the tool that switches between deployment chains can inject the correct WETH value here